packages/graphql/src/services/graphql.service.ts

```diff
@@ -98,7 +98,9 @@ export class GraphqlService implements BackendService {
     const columnIds: string[] = [];
     if (columnDefinitions && Array.isArray(columnDefinitions)) {
       for (const column of columnDefinitions) {
-        columnIds.push(column.field);
+        if (!column.excludeFieldFromQuery) {
+          columnIds.push(column.field);
```

Comment:
could you apply the same logic in the OData Service as well, they should have the same feature as much as possible